### PR TITLE
Addconfigmap

### PR DESCRIPTION
**Pull request by elizabethhalper**
Adding configmap values to helm chart and removing them from cluster deployment

### CODE REVIEW

File: LoginService/chart/values.yaml [hunk 1]
```diff
@@ -5,6 +5,14 @@
 appName: login-service
 env: production
 
+configmap:
+  aadAppId: 9b86d78b-3ee5-4502-a10b-46bd378b784f
+  appInsightsKey: a50ad6fc-04a4-41b2-b597-85965be4cd6c
+  jwtKey: olji4eoj30p8km4poij4mk520-jw3j
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply LoginService/chart/values.yaml]: I just spoke to Ivan, who said this value is now a secret and is no longer in config so I am removing it from values.yaml

File: LoginService/chart/README.md [hunk 2]
```diff
@@ -18,6 +18,27 @@ tiller-deploy-677436516-cq73w                   1/1       Running   0          2
 ...
 ```
 
+# Using Helm for Configuration Values
+This service uses Kubernetes configmaps in order to define configuration parameters. These values must be manually configured in the values.yaml file, and helm will be used to deploy the configmap, as well as the deployment and service. The steps to obtain these values are shown below.
```
Comment (billpratt) [inline LoginService/chart/README.md]: Link to values.yaml

File: LoginService/chart/README.md [hunk 3]
```diff
@@ -18,6 +18,27 @@ tiller-deploy-677436516-cq73w                   1/1       Running   0          2
 ...
 ```
 
+# Using Helm for Configuration Values
```
Comment (billpratt) [inline LoginService/chart/README.md]: Can you make the main header something like "Update Helm values" and make ConfigMap as a sub section? Also, please add a section here to update the follow values. I would put this section before ConfigMaps:

```app.image.repository```, ```app.image.imageName```, ```app.image.tag```, ```app.imagePullSecrets```, ```hexadite.image.repository```, ```hexadite.keyVault```

File: LoginService/chart/README.md [hunk 4]
```diff
@@ -18,6 +18,27 @@ tiller-deploy-677436516-cq73w                   1/1       Running   0          2
...
+This service uses Kubernetes configmaps in order to define configuration parameters. These values must be manually configured in the values.yaml file, and helm will be used to deploy the configmap, as well as the deployment and service. The steps to obtain these values are shown below.
+
+## Values to be Changed
+**aadAppId**
```
Comment (billpratt) [inline LoginService/chart/README.md]: Can you add instructions here stating that these values should be changed in the ```configs``` section in values.yaml?

**Pull request merged**